sklearn/mixture/_gaussian_mixture.py

```diff
@@ -314,20 +314,27 @@ def _compute_precision_cholesky(covariances, covariance_type):
         precisions_chol = np.empty((n_components, n_features, n_features))
         for k, covariance in enumerate(covariances):
             try:
-                cov_chol = linalg.cholesky(covariance, lower=True)
+                cov_chol = linalg.cholesky(covariance,
+                                           lower=True,
+                                           check_finite=False)
             except linalg.LinAlgError:
                 raise ValueError(estimate_precision_error_message)
             precisions_chol[k] = linalg.solve_triangular(cov_chol,
                                                          np.eye(n_features),
-                                                         lower=True).T
+                                                         lower=True,
+                                                         check_finite=False).T
     elif covariance_type == 'tied':
         _, n_features = covariances.shape
         try:
-            cov_chol = linalg.cholesky(covariances, lower=True)
+            cov_chol = linalg.cholesky(covariances,
```

Comment:
Same as https://github.com/scikit-learn/scikit-learn/pull/18850/files#r526344155

We do `covariance /= nk.sum()` line 194, so I would think we cannot assume that covariance will never be non finite ?

Reply:
Here `nk.sum()` is a positive integer so this operation is safe assuming `covariance` doesn't contain non-finite values initially.

Reply:
I do not think it is easy to see if `covariances` is finite here. I would revert.